wallet/how-to/manage-permissions.md

```diff
@@ -24,7 +28,13 @@ the [`eth_accounts`](/wallet/reference/eth_accounts) restricted method:
     </div>
 </div>
 
-## Example
+:::info Accessing accounts
+We recommend using [`eth_requestAccounts`](/wallet/reference/eth_requestAccounts), which
```

Comment:
Does this change make it easier for the user to understand?

"We suggest using eth_requestAccounts, which automatically asks for permission to use eth_accounts by invoking wallet_requestPermissions internally."